irods_client/irodsClient.py

```diff
@@ -0,0 +1,83 @@
+#from irods_client import _collections
+from irods_client.collection_operations import Collections
+from irods_client.data_object_operations import DataObjects
+from irods_client.query_operations import Queries
+from irods_client.resource_operations import Resources
+from irods_client.rule_operations import Rules
+from irods_client.ticket_operations import Tickets
+import requests
+
+class IrodsClient:
+    # Gets the username, password, and base url from the user to initialize a manager instance.
+    def __init__(self, url_base: str):
+        self.url_base = url_base
+        self.token = None
+
+        self.collections = Collections(url_base)
+        self.data_objects = DataObjects(url_base)
+        self.queries = Queries(url_base)
+        self.resources = Resources(url_base)
+        self.rules = Rules(url_base)
+        self.tickets = Tickets(url_base)
+
+    def authenticate(self, username: str='', password: str='', openid_token: str=''):
+        if (not isinstance(username, str)):
+            raise Exception('username must be a string')
+        if (not isinstance(password, str)):
+            raise Exception('password must be a string')
+        if (not isinstance(openid_token, str)):
+            raise Exception('openid_token must be a string')
+        
+        if (openid_token != ''): #TODO: Add openid authentication
+            return('logged in with openid')
```

Comment:
Why are parens wrapping the `return` values?

They return values don't appear to be treated as tuples.

Reply:
It's a habit that's kind of stuck over time, I can address thisas a part of housekeeping

Reply:
Please open an issue for this.

Reply:
Did you open an issue for this?

Reply:
#32 Opened a new one